site/projects/index.html

```diff
@@ -105,7 +105,7 @@ <h2 class="font-normal text-2xl-4xl">Text-UTF8 Migration</h2>
 
       <div class="space-y-4">
         <h2 class="font-normal text-2xl-4xl">Haskwrap</h2>
-        <p>The Haskell Foundation began with a conversation about the goals and and context for HF that changed as the conversation grew ever wider. As we resolved to launch HF this whitepaper is a snapshot of that conversation.</p>
+        <p>A Haskell library for installing Haskell toolchains, to serve as the de facto standard way to install GHC and other tools.</p>
```

Comment:
This sounds surprising to me -- I was expecting an executable, not a library. Maybe the idea is that Haskwrap is just the library while some other project will be the executable?

Reply:
Tbh I would have to check with Snoyman about what he means there, since I"m taking language directly from the proposal